docs/src/routes/docs/components/button.svx

```diff
@@ -0,0 +1,127 @@
+---
+shadow1: '0 2px 5px rgba(0, 0, 0, .14), 0 1px 5px rgba(0, 0, 0, .12);'
+shadowRaised: '0 2px 4px -1px rgba(0, 0, 0, .2), 0 3px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12);'
+---
+
+<script>
+  import { Button, Label } from 'attractions';
+  import { CheckIcon, XIcon } from 'svelte-feather-icons';
+  import H2 from 'src/mdsvex/h2.svelte';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+</script>
+
+# Button
+
+A clickable entity for all your controlling and linking needs.
+
+Comes in three flavours: filled, outline and flat.
+
+<Showcase>
+  <div slot="showcase">
+    <Button filled>button</Button>
+    <Button outline>button</Button>
+    <Button>button</Button>
+
+    <Button filled disabled>button</Button>
+    <Button outline disabled>button</Button>
+    <Button disabled>button</Button>
+
+    <Button filled danger>button</Button>
+    <Button outline danger>button</Button>
+    <Button danger noRipple>button</Button>
+
+    <Button filled>
+      <CheckIcon size="20" class="mr" />
+      icons
+    </Button>
+    <Button selected>selected</Button>
+    <Button round neutral><XIcon size="20" /></Button>
+  </div>
+  <div slot="source">
+
+```svelte
+<Button filled>button</Button>
+<Button outline>button</Button>
+<Button>button</Button>
+
+<Button filled disabled>button</Button>
+<Button outline disabled>button</Button>
+<Button disabled>button</Button>
+
+<Button filled danger>button</Button>
+<Button outline danger>button</Button>
+<Button danger noRipple>button</Button>
+
+<Button filled>
+  <CheckIcon size="20" class="mr" />
+  icons
+</Button>
+<Button selected>selected</Button>
+<Button round neutral><XIcon size="20" /></Button>
+```
+
+  </div>
+</Showcase>
+
+
+<H2 id="properties">Properties</H2>
+
+### Style Properties
+
+| <Label>Name</Label> | <Label>Type</Label> | <Label>Description</Label> |
```

Comment:
I have a strong feeling that this table can (should) be made into a component and just pass an array of data items

Reply:
Keep in mind that currently what we have is very expressive + retains full customizability. I'm quite hesitant to give up being able to throw an occasional component here and there. Besides, I see no way in which an array of data items representing a table could be formatted any better than a Markdown table